.env

```diff
@@ -53,7 +53,7 @@ DAM_SECRET=6a9e8a185b513363bc89ec0b96eed8f70c759bc86b97319f60365c4b7f8593dc
 # admin
 ADMIN_PORT=8000
 ADMIN_URL=http://localhost:$ADMIN_PORT
-SITES_CONFIG='{"main": {"url": "$SITE_URL", "preloginEnabled": false}, "secondary": {"url": "$SITE_URL", "preloginEnabled": false}}'
+SITES_CONFIG='{"main": {"url": "$SITE_URL", "preloginEnabled": false}, "secondary": {"url": "$SITE_PAGES_URL", "preloginEnabled": false}}'
```

Comment:
I decided that the secondary domain uses the Pages Router, which allows us to test the site preview for both the App Router and the Pages Router at the same time.